dependencies.props

```diff
@@ -9,20 +9,20 @@
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
   <PropertyGroup>
-    <CoreFxCurrentRef>892881e9575271257eb3f4c0106532dd3b9a41c3</CoreFxCurrentRef>
-    <CoreClrCurrentRef>892881e9575271257eb3f4c0106532dd3b9a41c3</CoreClrCurrentRef>
-    <CoreSetupCurrentRef>892881e9575271257eb3f4c0106532dd3b9a41c3</CoreSetupCurrentRef>
+    <CoreFxCurrentRef>23feda7eab2dc5d78f037c41afb6c2c8fc58914e</CoreFxCurrentRef>
+    <CoreClrCurrentRef>23feda7eab2dc5d78f037c41afb6c2c8fc58914e</CoreClrCurrentRef>
+    <CoreSetupCurrentRef>23feda7eab2dc5d78f037c41afb6c2c8fc58914e</CoreSetupCurrentRef>
     <ExternalCurrentRef>96dc7805f5df4a70a55783964ce69dcd91bfca80</ExternalCurrentRef>
-    <ProjectNTfsCurrentRef>892881e9575271257eb3f4c0106532dd3b9a41c3</ProjectNTfsCurrentRef>
-    <ProjectNTfsTestILCCurrentRef>892881e9575271257eb3f4c0106532dd3b9a41c3</ProjectNTfsTestILCCurrentRef>
+    <ProjectNTfsCurrentRef>23feda7eab2dc5d78f037c41afb6c2c8fc58914e</ProjectNTfsCurrentRef>
+    <ProjectNTfsTestILCCurrentRef>23feda7eab2dc5d78f037c41afb6c2c8fc58914e</ProjectNTfsTestILCCurrentRef>
     <SniCurrentRef>8bd1ec5fac9f0eec34ff6b34b1d878b4359e02dd</SniCurrentRef>
-    <StandardCurrentRef>270a80f37f01e09111b7b12858f1d0424bf9983b</StandardCurrentRef>
-    <BuildToolsCurrentRef>892881e9575271257eb3f4c0106532dd3b9a41c3</BuildToolsCurrentRef>
+    <StandardCurrentRef>23feda7eab2dc5d78f037c41afb6c2c8fc58914e</StandardCurrentRef>
+    <BuildToolsCurrentRef>23feda7eab2dc5d78f037c41afb6c2c8fc58914e</BuildToolsCurrentRef>
   </PropertyGroup>
 
   <!-- Product dependency versions. -->
   <PropertyGroup>
-    <NETStandardLibraryPackageVersion>2.0.4-servicing-26522-02</NETStandardLibraryPackageVersion>
+    <NETStandardLibraryPackageVersion>2.0.3</NETStandardLibraryPackageVersion>
```

Comment:
@weshaggard - what's the story with NETStandard.Library?  Is this correct?

Reply:
I moved it back to 2.0.3 as we aren't ready for any 2.0.4 versions yet, so this is expected.